docs/upgrade/index.rst

```diff
@@ -21,6 +21,7 @@ To ensure a successful upgrade, follow the :ref:`documented upgrade procedures <
 
 * You should perform the upgrades consecutively - to each successive X.Y version, **without skipping any major or minor version**.
 * Before you upgrade to the next version, the whole cluster (each node) must be upgraded to the previous version.
+* You cannot perform an upgrade by replacing the nodes in the cluster with new nodes with a different ScyllaDB version. You should never add a new node with a different release to a cluster.
```

Comment:
- We are using "a different ScyllaDB version" and "a different release", better use the same term.
- Need to exp[licitly mention this incudes patch versions